dist/theme-material.css

```diff
@@ -174,4 +174,63 @@
   margin-left: -5px;
   content: '';
   position: absolute;
+}
+
+.leaflet-control-layers-overlays {
```

Comment:
I am torn on whether I think this should be in a separate CSS file or not. maybe `ag-theme-material.css` and `leaflet-theme-material.css`? Don't want other devs to run into confusion if they have to edit this.